<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UseBasicColorsLesson</class>
 <widget class="QStackedWidget" name="UseBasicColorsLesson">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StackedWidget</string>
  </property>
  <widget class="QWidget" name="page">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Stencil</family>
     </font>
    </property>
    <property name="text">
     <string>Use 2-3 Basic Colors</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>311</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>When creating a flag, limit the number of colors on the flag to three, which contrast well and come from the standard color set. The basic flag colors are red, blue, green, black, yellow, and white. They can range from dark to light. Occasionally other colors are also used, such as purple, gray, and orange, but they are seldom needed in a good design.</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="nextButton_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>280</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="page_2">
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Stencil</family>
     </font>
    </property>
    <property name="text">
     <string>Use 2-3 Basic Colors</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>311</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>Separate dark colors with a light color, and light colors with a dark color, to help them create effective contrast. A good flag should also reproduce well in “grayscale”, that is, in black and white shades. More than four colors are hard to distinguish and make the flag unnecessarily complicated and expensive. Flag fabric comes in a relatively limited number of colors—another reason to stick to the basics.
</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>280</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
